br/pkg/restore/client.go

```diff
@@ -1374,7 +1400,7 @@ func (rc *Client) execChecksum(
 		ctx = opentracing.ContextWithSpan(ctx, span1)
 	}
 
-	startTS, err := rc.GetTS(ctx)
+	startTS, err := rc.GetTSWithRetry(ctx)
```

Comment:
can we replace all `rc.GetTS` with `rc.GetTSWithRetry`?

Reply:
Maybe yes. But `GetTS` is widely used, so I think it needs careful evaluation and testing.